modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -77,27 +77,29 @@ The `--copy-network` option only copies networking configuration found under `/e
 
 The disk partitions are created on {product-title} cluster nodes during the {op-system-first} installation. Each {op-system} node of a particular architecture uses the same partition layout, unless the default partitioning configuration is overridden. During the {op-system} installation, the size of the root file system is increased to use the remaining available space on the target device.
 
+Kubernetes supports monitoring `/var/lib/containers` when mounted in the same partition as `/var` or a separate partition. 
+
 There are two cases where you might want to override the default partitioning when installing {op-system} on an {product-title} cluster node:
 
-* Creating separate partitions: For greenfield installations on an empty
-disk, you might want to add separate storage to a partition. This is
-officially supported for mounting `/var` or a subdirectory of `/var`, such as `/var/lib/etcd`, on a separate partition, but not both.
+* Creating separate partitions: For greenfield installations on an empty disk, you might want to add separate storage to a partition. For example, mount `/var/lib/containers` in a separate partition, so that kubelet can monitor the `imagefs` directory for container runtimes,`/var/lib/containers` and the `nodefs` directory for root directory files.
 +
 [IMPORTANT]
 ====
-For disk sizes larger than 100GB, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" and this link:https://access.redhat.com/solutions/5587281[Red Hat Knowledgebase article] for more information.
+If you resized your disk size to host a larger file system, consider creating a separate `/var/lib/containers` partition. This is important for a disk formatted to `xfs`, where a high number of allocation groups might cause CPU time issues.
 ====
+// https://linux.die.net/man/5/xfs
```

Comment:
Hi @andreaskaris and @kannon92 , can I remove:

// https://linux.die.net/man/5/xfs
// https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/

They are commented out, but I'm not sure if they hold value and should be provided as links in the doc. Please let me know.

Reply:
I think so. I found it confusing also.

Reply:
Thanks, Removed.